doc/release_notes.md

```diff
@@ -14,15 +18,27 @@ on the PicoRV32 CPU and the CH552 means that device apps that have not
 been changed to use the protocol will not have any way to communicate
 with the outside world.
 
+### How to test TK1-Castor-alpha
+
+To test this alpha release you will need:
+
+- Tkey Unlocked
+- TKey Programmer Board
+- CH55x Reset Controller from [Blinkinlabs](https://shop-nl.blinkinlabs.com/products/ch55x-reset-controller)
+
+Read
+[here](https://github.com/tillitis/tillitis-key1/blob/main/README.md#building--flashing)
+for instructions.
+
 ### General
 
 - Split repo:
 
-    - tk1, mta1-usb-dev, mta-usb-v1 and mta1-library moves to
-      https://github.com/tillitis/tk1-pcba
+  - tk1, mta1-usb-dev, mta-usb-v1 and mta1-library moves to
+      <https://github.com/tillitis/tk1-pcba>
 
-    - tp1, mta1-usb-programmer, mta1-library and KiCad-RP Pico moves to
-      https://github.com/tillitis/tp1
+  - tp1, mta1-usb-programmer, mta1-library and KiCad-RP Pico moves to
+      <https://github.com/tillitis/tp1>
```

Comment:
Same as my comment above. Looks like there are 2 spaces too many at the start of the line.